### ISSUE TITLE
no apk builder

### ISSUE THREAD

**Issue by jbonn765** [bmshifat/TecSpy#11]
I looked everywhere but could not find the link to build the apk file.

**Comment by Lesbiathan** [comment]
yeah this is so cringe , just a copy and paste of L3MON code , probably malicious also.

**Comment by lat0s** [comment]
> I looked everywhere but could not find the link to build the apk file.

yeah this is so cringe , just a copy and paste of L3MON code , probably malicious also.
